ReactNativeTraining/src/app/screens/Home/screens/Chats/screens/Chat/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react';
```

Comment:
Like `Message` component, this component would be in src/app/screens/components/.. `Chat` component is going to need for Chats, Groups and Contancts chats